tests/test_fs.py

```diff
@@ -43,12 +51,12 @@ def test_ls(fs: V3ioFS, new_file, path_cls):
 
 
 def test_ls_with_marker(fs: V3ioFS, new_file):
-    for i in range(1200):
+    for i in range(5):
         new_file(fs._client, f"{test_dir}/test_ls/test-file{i}")
     path = str(f"/{test_container}/{test_dir}/test_ls")
 
-    out = fs.ls(path, detail=True)
-    assert len(out) == 1200, "not all files returned"
+    out = fs.ls(path, detail=True, limit=3)
+    assert len(out) == 5, "not all files returned"
```

Comment:
please delete test files and directories after running the test

Reply:
Existing test. I'll leave it to you to do in your aforementioned PR :)